### ISSUE TITLE
$iswindows ?

### ISSUE THREAD

**Issue by rogermartensson** [jonnyt/posh-itop#4]
Tried your latest release thinking I could add a cmdlet or two. 

You've already added those I found missing but I found a reference to a variable called $IsWindows.
That is part of PowerShell Core/6 and not anything earlier normally found on a windows installation.

Is it needed? If so, maybe a workaround should be made where $IsWindows is not available?

**Comment by jonnyt** [comment]
Yes that should be fixed then.  It's only needed if you are getting large result sets back because of limitations in the .Net implementation that I didn't run into when using PowerShell Core.  

This [discussion](https://github.com/PowerShell/PowerShell/issues/1280) has what looks like a good fix:

`$runningOnWindows = -not (Test-Path variable:IsWindows) -or $IsWindows`
